main/src/ca/uwaterloo/flix/language/phase/LambdaLift.scala

```diff
@@ -126,6 +126,14 @@ object LambdaLift {
       val es = exps.map(visitExp)
       LiftedAst.Expr.ApplyDef(sym, es, tpe, purity, loc)
 
+    case SimplifiedAst.Expr.ApplyLocalDef(sym, exps, tpe, purity, loc) =>
+      val es = exps.map(visitExp)
+      val newDefnSym = ctx.liftedLocalDefs.asScala.toMap.get(sym)
```

Comment:
Doing asScala.toMap.get sounds computationally expensive.

Reply:
Do you want a call to `find` instead?

Reply:
Just not convert the entire thing to a Scala map

Reply:
Find sounds fine

Reply:
If there is some null check involved you can add a helper method to SharedContext.

Reply:
I mean `cttx.liftedLocalDefs.asScala.find(...)` is ok?

Reply:
Without the as scala